functions/src/submitApplication/constraints.js

```diff
@@ -36,21 +54,15 @@ exports.constraints = {
     major: atLeastOneCharacter,
     phoneNumber: atLeastOneCharacter,
     ethnicity: atLeastOneCharacter,
-    // page two
+    // =========
+    //  PAGE TWO
+    // =========
     isFirstHackathon: {
       presence: true,
     },
-    // githubLink: {
-    //   url: true,
-    // },
-    // personalWebsiteLink: {
-    //   url: true,
-    // },
-    // linkedInLink: {
-    //   url: true,
-    // },
     resumeLink: {
       presence: true,
+      url: true,
     },
     interestForNwHacks: {
```

Comment:
hm these seem different from `PageThreeConstraints`? Can we update `PageThreeConstraints` to match this?